experimental/ag-ui/front-end/src/styles/query.css

```
@@ -0,0 +1,284 @@
+  font-size: 0.8rem;
+}
+
+.editor-action-btn:hover {
+  background: rgba(255, 255, 255, 0.06);
+  color: var(--text-primary);
+}
+
+.editor-action-btn svg {
+  width: 14px;
+  height: 14px;
+  stroke: currentColor;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)